src/app/modules/main/communities/models/curated_community_item.nim

```diff
@@ -44,6 +46,7 @@ proc initCuratedCommunityItem*(
   result.permissionModel = newTokenPermissionsModel()
   if tokenPermissionsItems.len > 0:
```

Comment:
just curious if the worth of checking permissions for banned user

Reply:
better to leave it... in case of any regression this is another "gate" not to show the user community content